### ISSUE TITLE
Mentorship opportunity

### ISSUE THREAD

**Issue by cthoyt** [MobleyLab/GuthrieSolv#6]
@davidlmobley I was thinking of how I would go about cleaning up/exploring the data in this repo now that there's a little infrastructure, but it occurred to me that this would be an excellent opportunity to mentor an excellent undergraduate or green graduate student on how practical data science goes (beyond using the iris dataset 😆) and a bit of machine learning. Let me know if someone immediately comes to mind, otherwise I will continue taking a crack at it myself

**Comment by davidlmobley** [comment]
Yeah, I don't have a perfect fit for this right now., in that mine are all doing this with other stuff somewhere else right now. BUT - perhaps science Twitter could serve?

**Comment by davidlmobley** [comment]
Note I DO typically have a very large tool of available undergrauates who have no expertise at all but would like to get involved in research somehow, so if we can identify concrete, bite-sized tasks that we need help with and spin them off into issues, I can get a bunch of rookies involved who have lots of enthusiasm but no experience. (Typically far more undergraduates want to do research with the group than I have room for, because of the lack of such tasks.) The timescale for bringing on someone new is probably the several to few weeks timescale once tasks are identified.

**Comment by cthoyt** [comment]
hi @davidlmobley, i just heard a talk from [euroSAMPL](https://qmbench.net/eurosampl) and your name came up! this had completely fell of my radar, maybe we can circle back to defining some concrete tasks on this

**Comment by davidlmobley** [comment]
Happy to touch bases; reach out by e-mail if you want to set up a call? I have little bandwidth to do someone myself, but am happy to help others do something.
